src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/gerritnotifier/ParameterExpander.java

```diff
@@ -381,12 +381,15 @@ protected Integer getVerifiedValue(Result res, GerritTrigger trigger) {
     /**
      * Returns the minimum verified value for the build results in the memory.
      * If no builds have contributed to verified value, this method returns null
-     * @param memoryImprint the memory.
-     * @param onlyBuilt only count builds that completed (no NOT_BUILT builds)
+     *
+     * @param memoryImprint    the memory.
+     * @param onlyBuilt        only count builds that completed (no NOT_BUILT builds)
+     * @param maxAllowedVerifiedValue Upper boundary on verified value.
      * @return the lowest verified value.
      */
     @CheckForNull
-    public Integer getMinimumVerifiedValue(MemoryImprint memoryImprint, boolean onlyBuilt) {
+    public Integer getMinimumVerifiedValue(MemoryImprint memoryImprint, boolean onlyBuilt,
```

Comment:
Normally a public method should not be changed for binary compatibility concerns, the old signature should be left intact but `@Deprecated` that just calls the new method with default values.
But I doubt that there are any usages of this class in other plugins, so I'll let it slide for now.

Reply:
Now when I think of it, maybe I should have written overloaded function instead and do the saturation there:

`
getMinimumVerifiedValue(MemoryImprint memoryImprint, boolean onlyBuilt,Integer maxAllowedVerifiedValue){
    return Math.min(maxAllowedVerifiedValue, getMinimumVerifiedValue(memoryImprint, onlyBuilt));
}
`